Lab1_Operationalizing_Pytorch_with_Mlflow/.labml.yaml

```diff
@@ -0,0 +1 @@
+experiments_path: 'labml_logs'
```

Comment:
Is this accidentally commited? Or will you be including labml parts in the project.  
(I'm all on for comparing labml training vs mlflow training btw, seems like visually labml is far superior)

Reply:
Sorry I was just using it while testing because i am familiar with it. I'll drop it and update. I didn't get a notification for your comment. I think I need to update my github communication preferences 😂

Reply:
It would be really nice to have both!

Reply:
Leaving it then